public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -531,7 +587,18 @@ export const AnomalyDetailsChart = React.memo(
                       }}
                     />
                   )}
-
+                  {/* <RectAnnotation
+                    dataValues={flattenData(generateContributionAnomalyAnnotations(zoomedAnomalies))}
+                    id="featureAttributionAnnotation"
+                    style={{
+                      stroke: CHART_COLORS.ANOMALY_GRADE_COLOR,
+                      strokeWidth: 1,
+                      opacity: 0.1,
+                      fill: CHART_COLORS.ANOMALY_GRADE_COLOR,
+                    }}
+                    renderTooltip={customAnomalyContributionTooltip}
+                  />   */}
+                
```

Comment:
I'm assuming this will be uncommented in future change. I'm ok to leave this as-is for now.